cmd/server/main.go

```diff
@@ -1,3 +1,15 @@
 package main
 
-func main() {}
+import (
+	"github.com/shestooy/go-musthave-metrics-tpl.git/internal/flags"
+	s "github.com/shestooy/go-musthave-metrics-tpl.git/internal/httpserver"
+)
+
+func main() {
+	flags.ParseServerFlags()
```

Comment:
```suggestion
	flags.ParseServer()
```
flags - уже есть, зачем повторять?

Reply:
тут немного не понял, что исправить